services/121-service/package.json

```diff
@@ -18,9 +18,11 @@
     "seed:dev": "ts-node src/scripts seed-dev",
     "test": "echo 'No automated tests. Run `npm run test:e2e:all` to run API-tests locally. Run `npm run test:unit:all` to run unit-tests locally.' && exit 0",
     "test:unit:all": "jest --runInBand --config=./jest.unit.config.js",
+    "test:unit:ci": "FORCE_COLOR=1 npm run test:unit:all -- --reporters='@matteoh2o1999/github-actions-jest-reporter'",
     "test:unit:watch": "npm run test:unit:all -- --watchAll",
     "test:unit:coverage": "npm run test:unit:all -- --coverage --coverageDirectory=coverage/unit",
     "test:e2e:all": "node --expose-gc node_modules/.bin/jest --config=jest.e2e.config.js --runInBand --detectOpenHandles --logHeapUsage",
+    "test:e2e:ci": "FORCE_COLOR=1 npm run test:e2e:all -- --reporters='@matteoh2o1999/github-actions-jest-reporter'",
```

Comment:
Apparently, judging by the screenshot, the "heap size" (i.e. memory use) isn't shown anymore for each test/scenario.
Is there a way to configure this reporter more?

And also, would it be possible to have it also working when running locally? (So to not have to many differences in the 'dev v.s. test' workflow/environments...

Reply:
> Apparently, judging by the screenshot, the "heap size" (i.e. memory use) isn't shown anymore for each test/scenario.
Is there a way to configure this reporter more?

Indeed, we seem to have lost that. I didn't notice. The reporter doesn't have any configuration options. Is this something we need/use in CI?

> And also, would it be possible to have it also working when running locally? (So to not have to many differences in the 'dev v.s. test' workflow/environments...

The reporter? We could, but it wouldn't make much sense given that it was built for CI (and the setup I've included is what is recommended by the author).

Reply:
Well, the heap-size isn't used "on a daily basis", but I think it is a good indicator of "badly written tests" ;)
So sometimes you can debug an issue based on "where the memory went".

> ... dev v.s CI

Some improvements to " make automated processes as smooth as possible" will improve both the 'development experience' AND the CI-flows...
As any developer will be running these tests locally mostly in an automated way (very similar to on GitHub)...